src/routes/(app)/expenses/upload/ExpenseReceipt.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  import FormFileInput from "$lib/components/forms/FormFileInput.svelte";
+  import {
+    arrayProxy,
+    type ArrayProxy,
+    type SuperForm,
+  } from "sveltekit-superforms";
+  import ExpenseReceiptRow from "./ExpenseReceiptRow.svelte";
+  import type { ExpenseSchema, ReceiptRowSchema } from "../types";
+  import { createBasicReceiptRow } from "../baseItem";
+
+  export let superform: SuperForm<ExpenseSchema>;
+  export let index: number;
+  export let onRemove: () => void;
+
+  const { values, errors } = arrayProxy(
+    superform,
+    `receipts[${index}].rows`,
+  ) as ArrayProxy<ReceiptRowSchema>;
+
+  let receiptPhotos: string[] | undefined = undefined;
+  const onFileSelected = async (
+    event: Event & {
+      currentTarget: EventTarget & HTMLInputElement;
+    },
+  ) => {
+    let images = event.currentTarget.files;
+    if (!images) return;
+    let reader = new FileReader();
+    receiptPhotos = [];
+    for (let image of images) {
+      await new Promise((resolve) => {
+        reader.readAsDataURL(image);
+        reader.onload = (e) => {
+          let result = e.target?.result ?? undefined;
+          // If array buffer, convert to base64
```

Comment:
använd `createImageBitmap` här istället

Reply:
I'm not sure. I've looked into createImageBitmap, which seems to require a canvas to draw on? I just took the code from the profile picture upload, which uses the reader. I tried going over to createImageBitmap but it did not play nicely with sveltekit (in the end, I do need to reference a canvas directly and draw on it). If I want to handle multiple images, it requires having a reference to the parent container and creating elements with javascript `.createElement(...)`, which goes against svelte.

If you want to fix this in a separate PR (for both this and profile picture upload).